corehq/apps/accounting/static/accounting/js/confirm_plan.js

```diff
@@ -14,7 +14,7 @@ var LIMITED_FEATURES = gettext("I need more limited features");
 var MORE_FEATURES = gettext("I need additional/custom features");
 var OTHER = gettext("Other");
 
-var confirmPlanModel = function (isUpgrade, isSameEdition, isPaused, currentPlan) {
+var confirmPlanModel = function (isUpgrade, isSameEdition, isPaused, isAnnualPlan, currentPlan) {
```

Comment:
confirm if `isSameEdition` and the related self variable is being used

Reply:
It's not! https://github.com/dimagi/commcare-hq/pull/36448/commits/bddb2bce26a9502fe95a0dabdeebf6d16ce73edd